src/voice_annotation_tool/opened_project_frame.py

```diff
@@ -50,10 +60,26 @@ def __init__(self):
         self.audioPlaybackWidget.previous_pressed.connect(self.previous_pressed)
         self.project: Project
         self.annotationList.installEventFilter(self)
+
         for age in AGE_STRINGS:
             self.ageInput.addItem(age)
         self.ageInput.addItem(self.tr("[Multiple]"))
 
+        self.recorder = QMediaRecorder()
+        self.input = QAudioInput()
+        self.session = QMediaCaptureSession()
+        self.session.setAudioInput(self.input)
+        self.recorder = QMediaRecorder()
+        self.session.setRecorder(self.recorder)
+        self.recorder.setMediaFormat(QMediaFormat.Wave)
+        self.recorder.setEncodingMode(QMediaRecorder.ConstantBitRateEncoding)
+        self.recorder.setAudioSampleRate(16000)
```

Comment:
I suppose to record and save in full quality, e.g. 48000 kHz, and then reduce the sample rate later for training and inference